src/with-translations.js

```diff
@@ -47,7 +47,7 @@ export const withTranslations = (
       }
     }
 
-    const displayName = Component.displayName || Component.name;
+    const displayName = Component.displayName || Component.name || 'Anonymous';
```

Comment:
In the commit description, but this is the same fix we applied here: https://github.com/fusionjs/fusion-react/pull/205/files